pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.github.org-arl</groupId>
     <artifactId>Fjage.jl</artifactId>
-    <version>0.2.0-SNAPSHOT</version>
+    <version>0.1.0</version>
```

Comment:
Why the downgrade here?

Reply:
Not a downgrade really. The number there is a notional number that maven needs to download. We don't need maintain a version number, but we do need to specify it for Maven. Hence just a notational `0.1.0`.